frontend/Dockerfile

```diff
@@ -12,15 +12,34 @@ FROM base AS shared
 
 WORKDIR /opt
 
-COPY shared/ ./shared
-RUN yarn workspace vaa-shared install --frozen-lockfile
-RUN yarn workspace vaa-shared build
+# Install all shared modules used by the frontend
```

Comment:
Adding all the shared modules here. We're not really using vaa-data yet, but it's gonna be used in the next update.